### PR TITLE
[Facades] Adds NS21 APIs

### PR DESCRIPTION
**Pull request by marek-safar**
Contributes to #11430

### CODE REVIEW

File: mcs/class/Facades/netstandard/apicheck.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```
Comment (akoeplinger) [inline mcs/class/Facades/netstandard/apicheck.sh]: is this file intentional included in this PR?

File: mcs/class/Facades/netstandard/TypeForwarders.cs [hunk 2]
```diff
@@ -2403,3 +2403,121 @@
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Xml.Xsl.XsltMessageEncounteredEventHandler))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Xml.Xsl.XslTransform))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Xml.Xsl.XsltSettings))]
+
+#if NS21
```
Comment (akoeplinger) [inline mcs/class/Facades/netstandard/TypeForwarders.cs]: why is this in an ifdef ?

File: mcs/class/Facades/netstandard/apicheck.sh (hunk 1 above)
Comment (marek-safar) [reply mcs/class/Facades/netstandard/apicheck.sh]: yes

**Comment by marek-safar** [comment]
@monojenkins commit csproj and apidiff
